recipe/meta.yaml

```
@@ -8,31 +8,33 @@ package:
 
 source:
   url: https://github.com/charman2/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 89457f747693a0fdd6c285401af274a2356aab69b3efaa54558d6d92a1cf4ee1
+  sha256: 4147f2da2e7c662970757ba7350dc98a1aeaf72c39c25a531b211dba2e126240
 
 build:
   number: 1
   skip: true  # [py==37]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy 1.24.*                           # [build_platform != target_platform]
+    - python  # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - numpy 1.24.*  # [build_platform != target_platform]
     - cmake
     - make  # [unix]
     - {{ compiler('m2w64-toolchain') }}  # [win]
     - {{ compiler('fortran') }}
     - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
```

Let's go ahead and include the C compiler here. Find that is generally a good idea when using C++ or Fortran

Also let's align quoting syntax in these Jinja functions either single...

```suggestion
    - {{ compiler('fortran') }}
    - {{ compiler('cxx') }}
    - {{ compiler('c') }}
    - {{ stdlib('c') }}
```

...or double

```suggestion
    - {{ compiler("fortran") }}
    - {{ compiler("cxx") }}
    - {{ compiler("c") }}
    - {{ stdlib("c") }}
```